tests/pkg/fixture/fixture.go

```diff
@@ -127,6 +127,77 @@ func GetTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
 	}
 }
 
+func GetFullTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
+	return &v1alpha1.TidbCluster{
```

Comment:
```
tc := GetTidbCluster()
tc.TiFlash = ...
```

get a basic tidb cluster with `GetTidbCluster`, then add TiFlash spec. This will simplify the code.

Reply:
Updated.